Formula/cgit.rb

```diff
@@ -0,0 +1,23 @@
+class Cgit < Formula
+  desc "Hyperfast web frontend for Git repositories written in C"
+  homepage "https://git.zx2c4.com/cgit/"
+  url "https://git.zx2c4.com/cgit/snapshot/cgit-1.1.tar.xz"
+  sha256 "0889af29be15fc981481caa09579f982b9740fe9fd2860ab87dff286f4635890"
+  depends_on "gettext"
+  depends_on "openssl"
+
+  def install
+    system "make", "get-git"
+    system "make",
```

Comment:
`system "make", "prefix=#{prefix}",` and then everything aligned with the `prefix` part is usually the style used.

Reply:
Those variables do not change when `prefix` changes. I have to manually specify them.

Reply:
I just mean stylistically this is the preferred way of writing this sort of thing _usually_:

```ruby
system "make", "prefix=#{prefix}",
               "CGIT_SCRIPT_PATH=#{pkgshare}"
```
and so on.